build.sh

```diff
@@ -4,6 +4,14 @@ set -e
 buildtags=""
 ldflags="-s -w"
 
+if [ "$1" = "bindata" ];then
+    GO_BINDATA_V=$(go-bindata -version | grep go-bin | cut -d ' ' -f2)
+    [[ "$GO_BINDATA_V" == "3.2.0" ]] && echo "Updating bindata" || exit "Wrong go-bindata-version"
```

Comment:
I think `[[` is a bashism and we're using `/bin/sh` here. How about using `[` instead?

Reply:
Oh, why do we want to use `/bin/sh`?

Reply:
Or put differently, you really have no guarantee what `/bin/sh` is going to be, so why not just use `#!/bin/bash` which is going to be present on most systems that are going to be used to build a CLI golang tool?

Reply:
I am not sure the extension change is really necessary as the shebang has changed. Anyways since the reason why I changed it is so small I will just use the more portable way of doing the same pattern and revert this.